apis/Google.Cloud.AspNetCore.DataProtection.SecretManager/.repo-metadata.json

```diff
@@ -0,0 +1,7 @@
+{
+    "distribution_name": "Google.Cloud.AspNetCore.DataProtection.SecretManager",
+    "release_level": "preview",
+    "client_documentation": "https://cloud.google.com/dotnet/docs/reference/",
```

Comment:
Is this url correct?

Reply:
No its not I was getting an error added it for debugging reasons. Forgot to remove it afterward will remove it. Thanks for pointing out. Right now we don't have any client docs for it.